packages/devkit/nx-reexports-pre16.ts

```diff
@@ -224,7 +224,7 @@ export {
 /**
  * @category Utils
  */
-export { Hash, Hasher } from 'nx/src/hasher/hasher';
+export { Hash, InProcessTaskHasher } from 'nx/src/hasher/task-hasher';
```

Comment:
This represents a breaking change in devkit, that breaks compatibility with Nx 15. For v16, we are committed to maintaining it. Maybe @FrozenPandaz has some ideas as to how to maintain it while making this rename?